reg_tests/global_cycle/driver.hercules.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+#-----------------------------------------------------------------------------
+#
+# Run global_cycle consistency tests on Hercules.
+#
+# Set $WORK_DIR to your working directory.  Set the project code and
+# and queue as appropriate.
+#
+# Invoke the script from command line as follows:  ./$script
+#
+# Log output is placed in consistency.log??.  A summary is
+# placed in summary.log
+#
+# A test fails when its output does not match the baseline files
+# as determined by the 'nccmp' utility.  This baseline files are
+# stored in HOMEreg.
+#
+#-----------------------------------------------------------------------------
+
+set -x
+
+source ../../sorc/machine-setup.sh > /dev/null 2>&1
+module use ../../modulefiles
+module load build.$target.intel
+module list
+
```

Comment:
To get this test to run for me I had to increase the stack size. After the module list, I added:

ulimit -s unlimited

All tests then ran successfully (and passed).

Reply:
Fixed, thanks!